app/assets/stylesheets/set_color.scss

```diff
@@ -2,3 +2,4 @@ $gray-medium-light: #eaeaea;
 $red: #ff0000;
```

Comment:
extract color code
it can be written briefer

Reply:
OK Anh